php/ClassInfo/ReflectionClass.php

```diff
@@ -4,27 +4,32 @@
 
 use PHPCD\PatternMatcher\PatternMatcher;
 
-class ReflectionClass extends \ReflectionClass implements ClassInfo
+class ReflectionClass implements ClassInfo
 {
     /**
      * @var PatternMatcher
      */
     private $pattern_matcher;
 
     /**
+     * @var \ReflectionClass
+     */
+    private $reflection;
+
+    /**
      * @param string|object $class
      * @param PatternMatcher $pattern_matcher
      */
     public function __construct($class, PatternMatcher $pattern_matcher)
     {
-        parent::__construct($class);
+        $this->reflection = new \ReflectionClass($class);
```

Comment:
If we inject it as a constructor parameter, where should we put the code to instance the `\ReflectionClass`?